tests/console/journalctl.pm

```diff
@@ -39,7 +39,7 @@ use constant {
 
 # by default persistent journal is set for opensuse, except of leap 15.3+
 sub has_default_persistent_journal {
-    return is_opensuse && !is_leap('>=15.3');
+    return is_opensuse && (!is_leap('15.3+') || is_leap('15.3+') && check_var('FLAVOR', 'JeOS-for-AArch64'));
```

Comment:
Previous `JeOS-for-AArch64` also have a persistent journal, so can be simplified to:

```suggestion
    return is_opensuse && (!is_leap('15.3+') || check_var('FLAVOR', 'JeOS-for-AArch64'));
```

Reply:
Perfect, just squash it into one commit. Thanks!